packages/graphql/src/graphql/scalars/Duration.ts

```diff
@@ -130,7 +130,11 @@ export const GraphQLDuration = new GraphQLScalarType({
             return value.toString();
         }
 
-        if (!DURATION_REGEX.test(value)) {
+        if (
+            !DURATION_REGEX_ISO.test(value) ||
+            !DURATION_REGEX_WITH_DELIMITERS.test(value) ||
+            !!DURATION_REGEX_NO_DELIMITERS.test(value)
```

Comment:
I think there may be an extra `!` here?
```suggestion
            !DURATION_REGEX_NO_DELIMITERS.test(value)
```

Reply:
Yes it was! Had to refactor this entire check here though